client/tooltips.js

```diff
@@ -0,0 +1,156 @@
+        "rtol * abs(y).  Value must be greater than 0.0.",
+
+    atol: "Absolute tolerance for ode solutions, if a component of y is approximately below atol, the error only needs to fall within the same atol threshold, and the "+
+        "number of correct digits is not guaranteed.  The solver keeps the local error estimates less than atol + rtol * abs(y).  Value must be greater than 0.0.",
+
+    ttol: "A relative error tolerance value governing tau-leaping tau selections.  Based on Cao, Y.; Gillespie, D. T.; Petzold, L. R. (2006). 'Efficient step size "+
+        "selection for the tau-leaping simulation method' (PDF). The Journal of Chemical Physics. 124 (4): 044109. Bibcode:2006JChPh.124d4109C. doi:10.1063/1.2159468. "+
+        "PMID 16460151  Value must be between 0.0 and 1.0.",
+
+    seed: "The seed for the simulation.  Set to -1 for a random seed.",
+
+    realizations: "The number of times to sample the chemical master equation. Each trajectory will be returned at the end of the simulation.",
+
+    algorithm:"The solver by which to simulate the model.",
+
+    ode: "This algorithm produces a deterministic continuous solution via ODE.",
+
+    ssa: "",
+
+    tauLeaping: "This algorithm calculates multiple reactions in a single step over a given tau step size.  The change in propensities over this step are bounded "+
+        "by bounding the relative change in state, yielding greatly improved run-time performance with very little trade-off in accuracy",
+
+    hybrid: "This algorithm uses a root-finding interpretation of the direct SSA method, along with ODE solvers to simulate ODE and Stochastic systems interchangeably "+
+        "or simultaneously.",
+
+    chooseForMe: "Hybrid will be chosen based on Event, Rules, and other SBML model components or if the model is represented as Concentration.  Tau Leaping will be "+
+        "chosen if the Tau Tolerance is changed to a value other than 0.03.  SSA will be chosen if the model is represented as Population."
+  },
+  parameterSweepSettings: {
+    sweepType: "The number of parameters to sweep through.",
+
+    species: "The initial species to view sweep data for.",
+
+    variable: "The parameter(s) to sweep through.",
+    
+    value: "The current value for the parameter you with to sweep through.",
+    
+    min: "The initial value of the sweep range.  Defaults to half of the current value.",
+    
+    max: "The final value of the sweep range.  Defaults to 1.5 times the current value.",
+    
+    steps: "The number of steps used to determining the sweep values across the sweep range."
+  },
+  parameterSweepResults: {
+    species: "The species to view sweep data for",
+
+    mapper: "The feature of the species population/concentration data for each sweep value.",
+    
+    reducer: "The type of aggregator to be applied to the trajectories of the species feature data for each sweep value."
```

Comment:
The statistical function applied to the ensemble of values extracted by the mapper from each simulation trajectory.